frontend/src/lib/Quiver.ts

```diff
@@ -599,30 +611,46 @@ but was expecting \`${JSON.stringify(expectedIndexTypes)}\`.
     return typeof indexName === "object" && indexName.kind === "range"
   }
 
+  /** Returns type for a single-index column or data column. */
+  public static getType(type: Type): IndexTypeName | string {
```

Comment:
Nit: rename to `getTypeName()`? (It's weird to have a function called `getType` that takes a `Type` object and returns a different type)